client-dotnet/src/Client.Util/Client.Util.csproj

```diff
@@ -5,15 +5,19 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     
     <PackageId>Client.Util</PackageId>
-    <Version>1.0.0-alpha</Version>
+    <Version>1.1.1-SNAPSHOT</Version>
     <Authors>EvoMaster Team</Authors>
     <Company>Kristiania</Company>
   </PropertyGroup>

```

Comment:
Hi,
  2 things here:
- in all these packages, should start with EvoMaster for the id, eg., `EvoMaster.Client.Util`. as i guess otherwise it will be a problem to upload those artifacts to NuGet in the future
- remove  `<Company>Kristiania</Company>`

Reply:
Hi, they are fixed now